lua-mode.el

```diff
@@ -36,7 +36,7 @@
 
 ;; This field is expanded to commit SHA, date & associated heads/tags during
 ;; archive creation.
```

Comment:
I guess we need to update the comment here as well then, right?
```suggestion
;; This field is expanded to commit SHA & date during archive creation.
```

Reply:
This is already done in my pr

Reply:
Thank you for the info!